### PR TITLE
This branch contains layout for desktop view and includes new image backgrounds

### PR DESCRIPTION
**Pull request by Solo7991**
Changes are:
New image backgrounds and patterns for desktop users
Introduction of Formspree forms in the contact section
New Layout for desktop
Navigation links in the welcome section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -267,55 +304,79 @@ <h3>LET'S CONNECT</h3>
...
+                            <form
+                                action="https://formspree.io/f/xayvljow"
+                                method="POST"
+                                class="flex"
+                            >
+                                <label for="name">
+                                    <input type="text" placeholder="Jack" id="name">
+                                </label>
+                                <label for="email">
+                                    <input type="text" placeholder="jack32@gmail.com" id="email">
```
Comment (keroloussamy) [inline index.html]: - [x] Kindly your email input should have `type="email"` not text kindly fix it.
- [x] Kindly your email input should have the `required` attribute.

File: index.html [hunk 2]
```diff
@@ -267,55 +304,79 @@ <h3>LET'S CONNECT</h3>
...
+                                </label>
+                                <label for="more"><textarea id="more"
+                                    >Write your message here</textarea>
+                                </label>
```
Comment (keroloussamy) [inline index.html]: - [x] Kindly your text area should have the `required` attribute.
- [x] Kindly your text area should have the `maxlength="500"` attribute.

File: index.html [hunk 3]
```diff
@@ -267,55 +304,79 @@ <h3>LET'S CONNECT</h3>
...
+                                needs coding. I’d love to help with it 
+                            </p>
+                        
+                            <form
+                                action="https://formspree.io/f/xayvljow"
+                                method="POST"
+                                class="flex"
+                            >
+                                <label for="name">
+                                    <input type="text" placeholder="Jack" id="name">
```
Comment (keroloussamy) [inline index.html]: - [x] Kindly your name input should have the `required` attribute.
- [x] Kindly your name input should have the `maxlength="30"` attribute.

File: index.html (hunk 2 above)
Comment (keroloussamy) [inline index.html]: - [ ] Kindly add `name` attribute to all your form elements, to specify each one is the name and the email, and the message. and to make your form submission works.

- [ ] Kindly be sure that your form is working before you submit it again.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Solo7991, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @Solo7991, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Solo7991** [comment]
This approval is for the desktop version of my portfolio. I created the form in the first week. Could I get approval for the "Add a contact form and finish the mobile version" project? How do I proceed?

**Comment by topeogunleye** [comment]
> This approval is for the desktop version of my portfolio. I created the form in the first week. Could I get approval for the "Add a contact form and finish the mobile version" project? How do I proceed?

Kindly create a new branch, then make a new pull request after you are done with your feature

**Pull request merged**